cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -205,6 +208,9 @@ type PostRequestBody struct {
 	from      *mail.Address
 	replyTo   *mail.Address
 	publicKey crypto.PublicKey
+	// encryption method to use
+	// required: false
+	Encryption byte `json:"encryption-method"`
```

Comment:
This is good, but also be great if we can have a more easily readable way of setting this value in addition to allowing to set the byte. I would propose  `encryption-method-name` which is the string value friendly name. This would then map to the byte values.

Reply:
Alright sure!

However, could you be more elaborate on what would happen if the both encryption-method and encryption-method-name is set to different values? Should I just throw an error or take precedence on one of it?

Reply:
Let make it that ONLY one of `encryption-method` or `encryption-method-name` can be set and throw an error otherwise. This is the simplest approach to explain to users through documentation.